clients/drcachesim/common/options.cpp

```diff
@@ -42,6 +42,20 @@ droption_t<bool> op_offline
  "If this option is enabled, trace data is instead written to files in -outdir "
  "for later offline analysis.  No simulator is executed.");
 
+droption_t<unsigned int> op_num_threads
+(DROPTION_SCOPE_CLIENT, "num_threads", 0, "Number of sideline threads for writing traces",
+ "For the offline analysis mode (when -offline is requested), specifies the number "
+ "of sideline threads to be used to write trace files out.  "
+ "Only 0 and 1 is currently supported.  "
+ "0 means synchronized write without any sideline threads.  ");
```

Comment:
I don't remember seeing any mention of this in the CL?  Did I miss it?  E.g., the scheme for putting an entry in the queue to terminate the sideline thread function isn't marked as temporary, so the reader can't tell why only one thread is supported and why we can't just add an array and for loops to easily increase the pool to 100 threads.